Cargo.toml

```diff
@@ -568,7 +571,7 @@ transforms-compound = []
 transforms-concat = []
 transforms-dedupe = ["lru"]
 transforms-field_filter = []
-transforms-filter = ["datadog-search-syntax", "vrl-parser", "vrl-compiler"]
+transforms-filter = []
```

Comment:
Previously, the `datadog_search` condition required the filter transform to be included.

This is technically incorrect, since a condition is a global construct. #10257 for context.